tests/deepsparse/transformers/pipelines/test_text_generation.py

```diff
@@ -51,6 +51,10 @@ def Fibonacci(n):
     "internal_kv_cache",
     [True, False],
 )
+@pytest.mark.parametrize(
```

Comment:
this is kind of ugly but makes do for now. we scan still quickly test the new chat scenario. refactor for the tests pending.